less/release-9.less

```diff
@@ -0,0 +1,309 @@
+        -webkit-box-shadow: -20px 25px 23px 0px rgba(156,156,156,1);
+        -moz-box-shadow: -20px 25px 23px 0px rgba(156,156,156,1);
+        box-shadow: -20px 25px 23px 0px rgba(156,156,156,1);
+
+        h3 {
+            padding-top: 70px;
+        }
+
+        p {
+            padding-bottom: 30px;
+        }
+
+        hr {
+            max-width: 150px;
+            border-top: 5px solid @brand-primary;
+            margin-left: 0;
+        }
+
+    }
+
+    .polygons {
+        background: @brand-secondary;
+        height: 75px;
+        position: relative;
+        width: 70px;
+
+    }
+
+    .polygons:after {
+        border-top: 10px solid @brand-secondary;
+        border-left: 35px solid transparent;
+        border-right: 35px solid transparent;
+        content: "";
+        height: 0;
+        left: 0;
+        position: absolute;
+        top: 75px;
+        width: 0;
+    }
+
+    .polygons-right {
+        float: right;
+        margin-top: -10px;
+    }
+
+}
+
+.jakartaee-hr {
+    max-width: 150px;
+    border-top: 5px solid @brand-primary;
+    margin-left: 0;
+}
+
+
+.font-numbers {
+
+    @media (min-width: @screen-sm-min) {
+      font-size: 100px;
+    }
+  
+    @media (max-width: @screen-xs-max) {
+      font-size: 65px;
+    }
+}
+
+.btn-watch-video {
+  &:hover {
+    background-color: @brand-primary;
+    border-color: @brand-primary;
+  }
+}
+
+.white-paper {
+  background-image: url("images/release/jakarta9-image.png");
+  background-repeat: no-repeat;
+  background-position: top;
+  background-size: cover;
+  overflow-x: hidden;
+  overflow-y: visible;
+}
+
+.white-paper-bg {
+  padding-left: 0;
+  padding-right: 0;
+  .btn-blue {
+    background-color: @white;
+    color: @brand-secondary;
+  }
+  .btn-blue:hover {
+    background-color: @brand-secondary;
+    color: @white;
+  }
+}
+
+.text-white {
+  color: @white;
+}
+
+.d-inline {
+  display: inline-block;
+}
+
+.white-paper-download {
+
+  @media (min-width: @screen-sm-min) {  // 768px
+      margin-top: 400px;
+      margin-bottom: 400px;
+  }
+
+  @media (max-width: @screen-xs-max) {  // 767px
+    margin-top: 200px;
+    margin-bottom: 200px;
+  }
+}
+
+.jakarta-download-social {
+  text-align:center;
+  padding:30px 0 50px;
+  .fa-stack {
+    color:@brand-primary;
+    &:hover {
+      color:@brand-secondary;
+    }
+  }
+}
+
+#footer-release-9 {
+  background: url("images/release/jakarta9-footer.jpg");
+  background-size: cover;
+  background-position: center;
+  background-repeat: no-repeat;
+  font-family: "Open Sans", "Helvetica Neue", Helvetica, Arial, sans-serif;
+  padding-top: 60px;
+  color: @white;
+  padding-bottom: 26px;
+  font-size: 14px;
+  border-top: none;
+  h2 {
+    color: @white;
+    margin-top: 0;
+    font-weight: 400;
+    font-size: 18px;
+  }
+
+  ul {
+    margin-left: -15px;
+  }
+
+  a {
+    color: @white;
+  }
+
+  a:hover {
+    background-color: transparent;
+  }
+
+  @media (max-width: @screen-xs-max) {
```

Comment:
Actually, for this one, since it's from footer style, after I merge the https://github.com/EclipseFdn/solstice-assets/pull/184 and https://github.com/EclipseFdn/hugo-solstice-theme/pull/179, I do not need it.